packaging/prover-proxy/postinst

```diff
@@ -21,6 +21,11 @@ then
 else
     mkdir -p /etc/opt/miden-prover-proxy
     sudo chown -R miden-prover-proxy /etc/opt/miden-prover-proxy
+    cat << EOF | sudo tee /etc/opt/miden-prover-proxy/service.conf
+    PORT=50051
+    PROVER_HOST=127.0.0.1
+    PROVER_PORT=50052
```

Comment:
This assumes that there is a single worker for the proxy, right? I wonder if we should just make it `WORKERS` and the value would be a list of workers in the format `HOST:PORT`. Or does this not work for some reason?

Also, do we not need to specify the type of the prover (tx vs batch vs block)?

Reply:
This should be covered by the env vars now.